bitnami/harbor/templates/chartmuseum/chartmuseum-svc.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.chartmuseum.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ template "harbor.chartmuseum" . }}"
+  labels:
+    {{ include "harbor.labels" . | nindent 4 }}
+    app.kubernetes.io/component: chartmuseum-service
+spec:
+  ports:
+    - port: 80
```

Comment:
Shouldn't this port be configurable?

Reply:
I think it is not necessary as it will be used by harbor-core and not by the end user. Any case, other services like `core/core-svc.yaml` are not making it configurable.

Reply:
Fair enough!